app/assets/src/api/index.js

```diff
@@ -224,7 +224,7 @@ const getSampleStats = ({ domain, filters, projectId, search }) =>
     },
   });
 
-const getSample = ({ sampleId }) => get(`/samples/${sampleId}/show_v2.json`);
+const getSample = ({ sampleId }) => get(`/samples/${sampleId}.json`);
```

Comment:
Do you also want to change the `getSampleReportData` and `getSampleReportInfo` methods earlier on in the file?

Reply:
You mean deleting the `getSampleReportInfo`? Yes.
What is your suggestion regarding `getSampleReportData` ?

Reply:
But I will probably delete on a future PR. At the same time I delete report_info on samples controller.